src/test/java/PlaySongTest.java

```diff
@@ -14,18 +15,21 @@ public class PlaySongTest extends BaseTest{
 
     @Test
     public void playSongTest() throws InterruptedException {
+        System.out.println("Im in playSongTest");
+
         login("demo@class.com", "te$t$tudent");
+
+        System.out.println("Im in playSongTestand I logged in");
+
         playSong();
         Assert.assertTrue(isSongPlaying());
         Thread.sleep(2000); // Wait for 10 seconds
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.